pkg/qdr/mgmt.go

```diff
@@ -80,9 +117,10 @@ func GetConnectedSites(edge bool, namespace string, clientset kubernetes.Interfa
 			if uplink == nil {
 				return result, nil
 			} else {
+
 				nodes, err := getNodesForRouter(uplink.Container, namespace, clientset, config)
 				if err == nil {
-					result, err := getConnectedSitesFromNodes(nodes, false, namespace, clientset, config)
+					result, err := getConnectedSitesFromNodesEdge(nodes, namespace, clientset, config)
```

Comment:
As above, the `nodes` passed in here are not needed or used in `getConnectedSitesFromEdge()`, so the `if edge` part of this should be completely replaced by the call to the new `getConnectedSitesFromEdge()`.

We should also get rid of the `getEdgeUplink()` method, which is now redundant (and was based on the incorrect assu,ption that there would be only one uplink).